### PR TITLE
Changed naming from Quantum Serverless to Qiskit Serverless

### PR DESCRIPTION
**Pull request by pandasa123**
## Background

As Serverless is integrating into the Qiskit stack, we want to rebrand Quantum Serverless --> Qiskit Serverless to match. This specific PR changes naming only, but we have a later PR to adjust how we talk about Serverless and Patterns, common configuration elements like dependency management, etc. 

## Merge Date

**Do not merge yet**

We have to finish the [Github / PyPi side of things for Qiskit Serverless](https://github.com/Qiskit-Extensions/quantum-serverless/issues/1307) before this PR is merged. We expect everything to be merged latest by **May 20th**

## Stakeholders

@Eric-Arellano, we'd need a redirect since the page URL is changing

### CODE REVIEW

**Comment by Eric-Arellano** [comment]
> We have to finish the https://github.com/Qiskit-Extensions/quantum-serverless/issues/1307 before this PR is merged. We expect everything to be merged latest by May 20th

We are going to be setting up redirects for every page ~this month as part of the docs reorganization. So we might be able to time these at the same time. Otherwise, I'll get the redirect set up using the current site structure.

**Comment by abbycross** [comment]
Hang on @pandasa123  - we don't want to change the URL of the Serverless page in IQP docs till the redirect is in place

File: docs/run-new/_toc.json [hunk 1]
```diff
@@ -79,8 +79,8 @@
       ]
     },
     {
-      "title": "Quantum Serverless workloads",
-      "url": "/run/quantum-serverless"
+      "title": "Qiskit Serverless workloads",
+      "url": "/run/qiskit-serverless"
```
[the remark this answers is not in the document]
Comment (abbycross) [reply docs/run-new/_toc.json:83]: keep the old URL for now until we are done with bigger re-org and have redirect in place

File: docs/run-new/index.mdx [hunk 2]
```diff
@@ -14,6 +14,6 @@ Step 3 in Qiskit Patterns, "Execute on target hardware," centers around the [Qis
 
 In addition to detailed information on primitives, you can read about the available execution modes in this section. While you can always submit a single job at a time, you can also use Runtime Primitives to run jobs in a [session](/run/sessions) or in [batch mode](/run/run-jobs-batch). These execution modes assist your utility-scale jobs to run efficiently. 
 
-This section also includes a page about our [processor types](/run/processor-types), how to [monitor jobs](/run/monitor-job) and [get information about our systems with the Qiskit SDK](/run/get-backend-information), details about [Quantum Serverless](/run/quantum-serverless), and more.
+This section also includes a page about our [processor types](/run/processor-types), how to [monitor jobs](/run/monitor-job) and [get information about our systems with the Qiskit SDK](/run/get-backend-information), details about [Qiskit Serverless](/run/qiskit-serverless), and more.
```
Comment (abbycross) [inline docs/run-new/index.mdx:17]:
```suggestion
This section also includes a page about our [processor types](/run/processor-types), how to [monitor jobs](/run/monitor-job) and [get information about our systems with the Qiskit SDK](/run/get-backend-information), details about [Qiskit Serverless](/run/quantum-serverless), and more.
```

Comment (abbycross) [reply docs/run-new/index.mdx:17]: Keeping old URL for now

File: docs/run/_toc.json [hunk 3]
```diff
@@ -94,8 +94,8 @@
       "url": "/run/visualize-results"
     },
     {
-      "title": "Quantum Serverless workloads",
-      "url": "/run/quantum-serverless"
+      "title": "Qiskit Serverless workloads",
+      "url": "/run/qiskit-serverless"
```
Comment (abbycross) [inline docs/run/_toc.json:98]:
```suggestion
      "url": "/run/quantum-serverless"
```

Comment (abbycross) [reply docs/run/_toc.json:98]: Keeping old URL

File: docs/run/quantum-serverless.mdx [hunk 4]
```diff
@@ -140,7 +140,7 @@ Then, in step 3 of the pattern, you execute the target circuit on the system cho
 
 from qiskit_ibm_runtime import QiskitRuntimeService, Session, Samplerv2 as Sampler
 from qiskit.circuit.random import random_circuit
-from quantum_serverless_tools.selectors import IBMLeastNoisyQPUSelector
+from qiskit_serverless_tools.selectors import IBMLeastNoisyQPUSelector
```
Comment (Eric-Arellano) [inline docs/run/quantum-serverless.mdx:143]: Is this a valid import?

**Comment by pandasa123** [comment]
@across, this is ready to go!

**Review by abbycross — APPROVED** [review]
👍

**Pull request merged**